share/man/man1/gpo.1

```diff
@@ -1,4 +1,4 @@
-.TH GPO "1" "March 2023" "gpodder 4.13.0" "User Commands"
+.TH GPO "1" "March 2023" "gpodder 4.13.2" "User Commands"
```

Comment:
Just like below, I'd rather see the version bump happen separately from this PR.